test/inductor/test_smoke.py

```diff
@@ -24,6 +25,7 @@ def _test_f(x):
 
 
 class SmokeTest(TestCase):
+    @unittest.skipIf(IS_JETSON, "openAI Triton not available for Jetson")
```

Comment:
This should just skip based on  (which just checks whether triton is installed or not)
https://github.com/pytorch/pytorch/blob/641cb4243c2ed54b02e713d03659824034b6c82f/torch/testing/_internal/inductor_utils.py#L23
```suggestion
    @unittest.skipIf(torch._internal.inductor_utils.HAS_CUDA, "Triton is not available")
```

Reply:
https://github.com/pytorch/pytorch/actions/runs/4197589008/jobs/7280465927
your suggested change is failing CI. should i revert to my original IS_JETSON skipIf or is there some quick fix for the failure ?